civiclens/collect/qa_doc_comment_num.py

```diff
@@ -0,0 +1,120 @@
+import requests
+import time
+import polars as pl
+from civiclens.collect.move_data_from_api_to_database import connect_db_and_get_cursor
+
+from civiclens.utils.constants import (
+    REG_GOV_API_KEY,
+)
+
+
+def pull_list_of_doc_info() -> list[tuple[str]]:
+    connection, cursor = connect_db_and_get_cursor()
+    with connection:
+        with cursor:
+            query = "SELECT id, object_id, rin \
+                    FROM regulations_document \
+                    WHERE open_for_comment = True;"
+            cursor.execute(query)
+            response = cursor.fetchall()
+
+    return response
+
+
+def find_object_id(object_id, rin):
```

Comment:
Not a major issue, but I wonder why rin is used?

Reply:
This is to address the misalignment in our data columns -- it looks like prior to mid April the object id was actually placed in the rin field. So find_object_id extracts the right value from one of those fields